database_admin/schema/create_schema.sql

```diff
@@ -935,13 +935,13 @@ CREATE TABLE IF NOT EXISTS system_package
     -- Use null to represent up-to-date packages
     update_data   JSONB DEFAULT NULL,
     latest_evra   TEXT GENERATED ALWAYS AS ( ((update_data ->> -1)::jsonb ->> 'evra')::text) STORED,
-    name_id       INTEGER REFERENCES package_name (id) NOT NULL,
+    name_id       INTEGER REFERENCES package_name (id) ,
 
     PRIMARY KEY (rh_account_id, system_id, package_id) INCLUDE (latest_evra)
 ) PARTITION BY HASH (rh_account_id);
 
-CREATE INDEX IF NOT EXISTS system_package_name_pkg_system_idx
-    ON system_package (rh_account_id, name_id, package_id, system_id) INCLUDE (latest_evra);
+--CREATE INDEX IF NOT EXISTS system_package_name_pkg_system_idx
```

Comment:
I don't know why are you removing the index.

Reply:
It depends on the column definition. We are adding and removing Not null constraint to the column

Reply:
That means we will have slightly different schemas in ci/qa/stage and in prod.